src/dapp.js

```diff
@@ -34,10 +13,10 @@ export function getContract() {
         web3 = new Web3();
     }
     web3.setProvider(new web3.providers.HttpProvider(provider));
-    var account = getAccountAddress();
-    web3.eth.defaultAccount = '0x' + account;
+    var account = web3.eth.accounts[0]
+    web3.eth.defaultAccount = account;
     console.log("Account " + account)
-    var address = "5ebf511899003855c89af07acf6df7d437ad4cf8"
+    var address = contractAddress
```

Comment:
where does this come from?

Reply:
maybe should be some kind of garbage value so it doesn't compile and people know when they've not configured it as it blows up in a spectactular fashion.

Reply:
ooh, or a comment
`// don't forget to change this to your own contract address!!!`
it does need the three exclamation points.